hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/NodeDecommissionManager.java

```diff
@@ -398,10 +398,12 @@ private synchronized boolean checkIfDecommissionPossible(List<DatanodeDetails> d
         if (opState != NodeOperationalState.IN_SERVICE) {
           numDecom--;
           validDns.remove(dn);
+          LOG.warn("Cannot decommission {} because it is not IN-SERVICE", dn.getHostName());
```

Comment:
I think this message (and similar one for maintenance) could be improved by including the actual `opState` the node is in.

Also, please consider replacing `dn.getHostName()` with `dn` to rely on:

https://github.com/apache/ozone/blob/f784a84a04bbabde01f3154d24a1eb92344088d1/hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/protocol/DatanodeDetails.java#L561-L563

This shows just a bit more info to make the node uniquely identifiable while keeping the hostname for convenience.  (Applies to all similar logs.)